index.js

```diff
@@ -77,9 +77,11 @@ if (argv.gasprice < 1 || argv.gasprice > 1000) throw "--gasprice must be between
   };
 
   const configSettings = {
-    rewardRatePerSecond: { rawValue: "0" },
-    proposerBondPercentage: { rawValue: "0" },
+    rewardRatePerSecond: { rawValue: toWei("0.000001") }, // 1% every 10_000 seconds
```

Comment:
Isn't that really high? Wouldn't this be a ~10% fee per day?

Reply:
So does `0.03` translate to `toWei(0.0000000009645)`?

`0.03/360/24/60/60`